source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -78,6 +84,37 @@ closes if there is an early termination:
 
 For more information on how to ensure a cursor closes, see the :ref:`cursor cleanup section <cursor_cleanup>`.
 
+Explain
+~~~~~~~
+
+Use the ``explain()`` method to determine the execution plan of an
+operation with the given verbosity level: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Cursor.java
+   :language: java
+   :dedent:
+   :start-after: begin explainExample
+   :end-before: end explainExample
+
+Your output should look something like the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [Document{{$cursor=Document{{queryPlanner=Document{{plannerVersion=1,
+   ...
+   winningPlan=Document{{stage=PROJECTION_SIMPLE, transformBy=Document{{stars=1, _id=0}}, 
+   inputStage=Document{{stage=COLLSCAN, filter=Document{{categories=Document{{$eq=Bakery}}}}, 
+   direction=forward}}}}, rejectedPlans=[]}}, executionStats=Document{{executionSuccess=true, 
+   nReturned=3, executionTimeMillis=0, 
+   ...
+
+.. include:: /includes/fundamentals/explain-verbosity.rst
+
+For more information about this section, see the
+:java-docs:`explain() <apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`
+method API documentation.
```

Comment:
Suggestion: As written, I think the reader has to read the whole sentence in order to gain context for the link. I believe that this means the reader has to read the whole sentence to get context and then back track to click the link.

[From the style guide:](https://docs.mongodb.com/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/)
>  Don’t make users backtrack to understand a sentence. If they can’t understand the beginning of the sentence without reading the end, rewrite the sentence.

```suggestion
API documentation for the 
:java-docs:`explain() method <apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`.
```